user_tools/src/spark_rapids_tools/cmdli/tools_cli.py

```diff
@@ -56,15 +55,15 @@ def qualification(self,
         The cmd will process each app individually, but will group apps with the same name into the
         same output row after averaging duration metrics accordingly.
 
-        :param platform: Defines one of the following: "onprem", "emr", "dataproc", "dataproc-gke",
-               "databricks-aws", and "databricks-azure".
         :param eventlogs: Event log filenames or CSP storage directories containing event logs
                 (comma separated).
 
                 Skipping this argument requires that the cluster argument points to a valid
                 cluster name on the CSP.
         :param cluster: The CPU cluster on which the Spark application(s) were executed.
                Name or ID (for databricks platforms) of cluster or path to cluster-properties.
+        :param platform: Defines one of the following: "onprem", "emr", "dataproc", "dataproc-gke",
```

Comment:
do we still support `dataproc-gke`?

Reply:
That's a good point @cindyyuanjiang 
We might need to have a separate issue to clean-up dataproc-gke and decide how shall we handle platform-variations.
For example, dataproc vs dataproc-gke and AWS vs. EMR...etc

Reply:
@amahussein understood, thanks!

Reply:
Thanks @cindyyuanjiang. I can file another issue to drop `gke` as supported platform.